### PR TITLE
[17.0][MIG] product_supplierinfo_archive

### PR DESCRIPTION
**Pull request by CarlosVForgeFlow**
Migration to 17.0

@ForgeFlow

### CODE REVIEW

File: odoo.conf
Comment (LoisRForgeFlow) [inline odoo.conf:1]: This file does not belong here I think :sweat_smile:

**Review by JordiMForgeFlow — APPROVED** [review]
Code review LGTM

File: odoo.conf
Comment (CarlosVForgeFlow) [reply odoo.conf:1]: You're right, it's not there anymore. Thanks

**Review by MiquelRForgeFlow — APPROVED** [review]
Please squash last 2 commits.

**Pull request merged**